fail2ban/server/server.py

```diff
@@ -324,6 +324,15 @@ def setUnbanIP(self, name, value):
 	def getBanTime(self, name):
 		return self.__jails[name].actions.getBanTime()
 	
+	def isAlive(self, jailnum=None):
+		if jailnum is not None and len(self.__jails) != jailnum:
+			return 0
+		for j in self.__jails:
+			j = self.__jails[j]
```

Comment:
no, it's correct so, and without it it's broken (__jails is a "dict")... and it's covered by tests:
```
ERROR: testStartStopJail (fail2ban.tests.servertestcase.Transmitter)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "./fail2ban/tests/servertestcase.py", line 245, in testStartStopJail
    3) )
  File "./fail2ban/server/utils.py", line 192, in wait_for
    while not cond():
  File "./fail2ban/tests/servertestcase.py", line 244, in <lambda>
    lambda: self.server.isAlive(1) and not isinstance(self.transm.proceed(["status", self.jailName]), RuntimeError),
  File "./fail2ban/server/server.py", line 331, in isAlive
    if not j.isAlive():
AttributeError: 'str' object has no attribute 'isAlive'
```
but I will rewrite it with `__jails.values()` :
```python
		for jail in self.__jails.values():
			if not jail.isAlive():
				return 0
```
Why should I prove for you the accuracy of each line? A little trust must be also, or not?